Use case of UDR with Express Route and BGP

FabianKLS opened the pull request:
In relation to BGP: The original text states "You cannot create user-defined routes if you deploy a virtual network gateway deployed as type: ExpressRoute". This isn't correct. You still can use UDR to force traffic from the ER GW to, for example, an NVA. This is inline with Microsofts reference architectures. It isn't possible to define a User Defined Route to force traffic back to onprem. This is handled by the BGP advertised routes.

azurecla commented:
Hi __@olegsych__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

azurecla commented:
__@olegsych__, Thanks for signing the contribution license agreement so quickly! Actual humans will now validate the agreement and then evaluate the PR.
<br />Thanks, AZPRBOT;

alancameronwills commented:
I created a new PR for these changes. Please close this one.

olegsych commented:
Could you reference the new PR here? 
Did I do something wrong with this one?

alancameronwills commented:
https://github.com/Azure/azure-content-pr/pull/12087

Just that it seemed to be taking a long time for this to go through. I just copied all your changes, apart from a spare "."

FabianKLS committed:
Use case of UDR with Express Route and BGP

In relation to BGP: The original text states "You cannot create user-defined routes if you deploy a virtual network gateway deployed as type: ExpressRoute". This isn't correct. You still can use UDR to force traffic from the ER GW to, for example, an NVA. This is inline with Microsofts reference architectures. It isn't possible to define a User Defined Route to force traffic back to onprem. This is handled by the BGP advertised routes.
